tuning/scripts/analyze_results.sh

```diff
@@ -0,0 +1,125 @@
+#!/bin/sh
+
+HELP=false
+PLOT=true
+MFMA=false
+COUNT=false
+
+HELP_STR="
+Usage: ${0} WORKING_PATH LOG_PATH LIBRARY [options]
+
+  where LIBRARY = {arcturus | vega20 | vega10 | mi25 | r9nano | hip}
+
+Options:
+-h | --help             Display this help message
+-s | --size             Data size
+-f | --sclk             Frequency of sclk in MHz
+-m | --mfma             Was MFMA enabled
+-c | --count            Sets all cases where count=1 to count=10
+-n | --no-plot          Skip plotting
+"
+
+if ! OPTS=$(getopt -o h,s:,f:,m,c,n \
+--long help,size:,sclk:,mfma,count,no-plot -n 'parse-options' -- "$@")
+then
+  echo "Failed parsing options"
+  exit 1
+fi
+
+eval set -- "${OPTS}"
+
+while true; do
+  case ${1} in
+    -h | --help )              HELP=true; shift ;;
+    -s | --size)               SZ=${2}; shift 2;;
+    -f | --sclk)               SCLK=${2}; shift 2;;
+    -m | --mfma )	             MFMA=true; shift ;;
+    -c | --count )	           COUNT=true; shift ;;
+    -n | --no-plot )           PLOT=false; shift ;;
+    -- ) shift; break ;;
+    * ) break ;;
+  esac
+done
+
+if ${HELP}; then
+  echo "${HELP_STR}"
+  exit 0
+fi
+
+if [ -z ${SZ+x} ]; then
+  echo "Data size not specified: assuming 4 bytes"
+  SZ=4
+fi
+
+if [ -z ${SCLK+x} ]; then
+  echo "Clock rate not specified: assuming 1000 MHz"
+  SCLK=1000
+fi
+
+if [ $# != 3 ]; then
+  echo "Exactly three possitional args required"
+  echo "See ${0} --help"
+fi
+
+WORKING_PATH=${1}
+LOG=${2}
```

Comment:
The current version of this has a log file as a requirement. This was not always the case. This log file is use to extract the call count information. It would be nice if this was optional, in which case the call counts would be set to 1 by default. 

This would enable a user to run the analysis apart from the context of some application use case.

Reply:
At some point the log file was made mandatory, and I left it that way for now as PerformanceAnalysis.py requires it. I agree this should be made optional, but might be outside the scope of this PR.